src/dda/cli/self/features/enabled/__init__.py

```diff
@@ -0,0 +1,61 @@
+# SPDX-FileCopyrightText: 2025-present Datadog, Inc. <dev@datadoghq.com>
+#
+# SPDX-License-Identifier: MIT
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import click
+
+from dda.cli.base import dynamic_command, pass_app
+
+if TYPE_CHECKING:
+    from dda.cli.application import Application
+
+
+def _parse_scopes(values: tuple[str, ...]) -> dict[str, str]:
+    scopes: dict[str, str] = {}
+    for item in values:
+        if "=" not in item:
+            err_message = f"Invalid scope '{item}', expected key=value"
+            raise click.BadParameter(err_message)
+        key, value = item.split("=", 1)
+        key = key.strip()
+        if not key:
+            err_message = f"Invalid scope '{item}', key cannot be empty"
+            raise click.BadParameter(err_message)
+        scopes[key] = value
+    return scopes
+
+
+@dynamic_command(short_help="Evaluate a feature flag")
+@click.argument("flag", metavar="FLAG", type=str)
```

Comment:
This is the default behavior.


```suggestion
@click.argument("flag")
```